### ISSUE TITLE
Enh: additional weave networks

### ISSUE THREAD

**Issue by hans-d** [kontena/kontena#343]
Would like to create some additional networks within a grid, and be able to connect a container to one a more such networks.

Eg for seperate networks for dbs, apps, or for splitting a staging environment within the same grid.

**Comment by jakolehm** [comment]
Do you mean that each service (or app) could belong to multiple networks? For example:
```
db-network: 10.82.0.0/16
app-network: 10.83.0.0/16
```
In this case database backed app must be in `app-network` and also `db-network` (because app needs connection to db-network).

**Comment by hans-d** [comment]
A loadbalancer is a good candidate for bridging in some of thse cases.

**Comment by jakolehm** [comment]
Ok, so in my previous example there would be `db-loadbalancer` that belongs to both networks and app connects to database through that... sounds pretty good.

**Comment by miskun** [comment]
Yes, i like this! In fact, there has been now several people looking for something like this so I assume it is needed.

**Comment by faddat** [comment]
I'd like to +1 this also-- the networking in our deploys keeps getting more complex-- at a rather fast pace, too.

**Comment by mtrofimm** [comment]
:+1:

**Comment by mtrofimm** [comment]
@miskun any update on this issue?

**Comment by miskun** [comment]
@mtrofimm Hey! Yes, it's now included in the Kontena roadmap and I hope we can start working on this soon. @jakolehm is still tweaking some final details on the roadmap, but I hope we can publish it soon.

**Comment by mtrofimm** [comment]
@miskun thanks for information. 
One more thing: any plans to implement configuration of the weave subnet block while creating grid? I mean I would like to integrate grids without ip conflicts. The simple solution would be to change the default subnet 10.81.0.0/16 while creating grid. 
It's possible in convox to define subnets while installing rack into aws:
```
 --vpc-cidr "10.0.0.0/16"		The VPC CIDR block
   --subnet0-cidr "10.0.1.0/24"		Subnet 0 CIDR block
   --subnet1-cidr "10.0.2.0/24"		Subnet 1 CIDR block
   --subnet2-cidr "10.0.3.0/24"		Subnet 2 CIDR block
   --subnet-private0-cidr "10.0.4.0/24"	Private Subnet 0 CIDR block
   --subnet-private1-cidr "10.0.5.0/24"	Private Subnet 1 CIDR block
   --subnet-private2-cidr "10.0.6.0/24"	Private Subnet 2 CIDR block
```

**Comment by mtrofimm** [comment]
@miskun any news about ETA?
